### DISCUSSION TITLE
Map rendering needs more contrasr when mapping outdoor

### DISCUSSION THREAD

**Discussion by dtuser007** [streetcomplete/StreetComplete#5688] [Feedback]
great app, but display of map is weak compared with osmand.
regards

**Comment by mnalis** [comment]
Are you using dark or light mode?
Can you give us an example (with screenshot?) of what you mean by _"more contrasr"_? 

Background map in StreetComplete is intentionally kept simple without too much details (in order to not overwhelm the users and for quest icons to stand out), if you meant that by _"weak"_.

**Comment by dtuser007** [comment]
Hi,
I am using dark mode usually due to better battery life. 
I investigated the issue a bit more: I guess in dark mode the loss of colours makes it more difficult to link the map to the landscape and buildings. I'll try light mode next time when mapping...
It is not the number of icons that concerns me, but the necessity to get a clear and quick comprehension, where in reality the todos are located.
